internal/mempool/mempool.go

```diff
@@ -869,7 +874,21 @@ func (mp *TxPool) removeTransaction(tx *dcrutil.Tx, removeRedeemers bool, isTrea
 		for _, txIn := range txDesc.Tx.MsgTx().TxIn {
 			delete(mp.outpoints, txIn.PreviousOutPoint)
 		}
+
+		// Stop tracking this transaction in the mining view.
+		updateDescendentStats := true
```

Comment:
I see this all throughout, but the preferred spelling is `descendant`.  `Descendent` is more of a historical usage that was used as an adjective to mean "going down".  This is a commonly confused word pair and is misused often enough that it is sometimes considered a less commonly used variant of the correct form `descendant` in this context, which is the opposite of an ancestor.